### PR TITLE
Created empty app

### PR DESCRIPTION
**Pull request by katiefenn**

### CODE REVIEW

File: src/common/containers/app.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes, Component } from 'react';
+import Helmet from 'react-helmet';
+
+export class App extends Component {
+  render() {
+    return (
+      <div>
+        <Helmet
+          htmlAttributes={{ 'lang': 'en' }}
+          titleTemplate='build.snapcraft.com - %s'
```
Comment (earnubs) [inline src/common/containers/app.js]: snapcraft.io

File: src/common/containers/app.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          defaultTitle='build.snapcraft.com'
+          meta={[
+            { 'name': 'description', 'content': 'build.snapcraft.com ui' },
```
Comment (earnubs) [inline src/common/containers/app.js]: no need for 'ui', suggest just build.snapcraft.io?

File: src/server/handlers/universal.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        delete req.session.error;
+      }
+
+    }
+
+    const store = configureStore(initialState);
+
+    // config we share from server side to client side
+    const config = {
+      UNIVERSAL: conf.get('UNIVERSAL')
```
Comment (bartaz) [inline src/server/handlers/universal.js]: I think we wanted to get rid of UNIVERSAL/SERVER prefixes in config.
But maybe it's something for another branch.

File: src/server/helpers/html.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+          {/*
+            Can't add script to Helmet in here (as we are already rendering <head>).
+            And having script added to Helmet in App causes script to be loaded
+            twice (once from server side rendered HTML, once on client side).
+
+            See more:
+            https://github.com/nfl/react-helmet/issues/149
+            https://github.com/canonical-ols/javan-rhino/issues/176
+          */}
+          <script src='https://js.stripe.com/v2/'></script>
```
Comment (katiefenn) [inline src/server/helpers/html.js]: Good spot 👍

**Pull request merged**